docs/content/_navigation.json

```diff
@@ -5,439 +5,195 @@
     "path": "/getting-started",
     "children": [
       {
-        "title": "Installation",
-        "path": "/getting-started/install"
+        "title": "What is Dagster?",
+        "path": "/getting-started/what-is-dagster"
       },
       {
-        "title": "Create a New Project",
-        "path": "/getting-started/create-new-project"
+        "title": "Architecture",
+        "path": "/getting-started/architecture"
       },
       {
         "title": "Telemetry",
         "path": "/getting-started/telemetry"
       },
       {
-        "title": "Releases and Changes",
-        "path": "/getting-started/releases"
-      }
-    ]
-  },
-  {
-    "title": "Tutorials",
-    "icon": "AcademicCap",
-    "path": "/tutorial",
-    "children": [
-      {
-        "title": "Setup",
-        "path": "/tutorial/setup"
-      },
-      {
-        "title": "Assets",
-        "children": [
-          {
-            "title": "Defining an Asset",
-            "path": "/tutorial/assets/defining-an-asset"
-          },
-          {
-            "title": "Building Graphs of Assets",
-            "path": "/tutorial/assets/asset-graph"
-          },
-          {
-            "title": "Testing Assets",
-            "path": "/tutorial/assets/testing-assets"
-          }
-        ]
+        "title": "Installation",
+        "path": "/getting-started/installation"
       },
       {
-        "title": "Ops and Jobs",
-        "children": [
-          {
-            "title": "Writing, Executing a Single-Op Job",
-            "path": "/tutorial/ops-jobs/single-op-job"
-          },
-          {
-            "title": "Connecting Ops In Jobs",
-            "path": "/tutorial/ops-jobs/connecting-ops"
-          },
-          {
-            "title": "Testing Ops and Jobs",
-            "path": "/tutorial/ops-jobs/testable"
-          }
-        ]
+        "title": "Creating a project",
+        "path": "/getting-started/create-new-project"
       }
     ]
   },
   {
-    "title": "Concepts",
+    "title": "Building with Dagster",
     "icon": "Menu",
-    "path": "/concepts",
+    "path": "/building",
     "isDefaultOpen": true,
     "children": [
       {
-        "title": "Assets",
+        "title": "Understanding Dagster",
         "children": [
           {
-            "title": "Software-Defined Assets",
-            "path": "/concepts/assets/software-defined-assets"
+            "title": "Designing DAGs",
+            "path": "/building#designing-dags"
```

Comment:
I find it a little confusing to have some left nav items take you to subsections and others take you to standalone pages.